core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -11032,6 +11033,43 @@ private RelNode createRelNodeWithQualifyStatement() {
   private static RelNode toLogical(RelNode rel, RelBuilder builder) {
     return rel.accept(new ToLogicalConverter(builder));
   }
+
+  @Test public void testSubQueryWithInClauseRule() {
```

Comment:
Please rename the test case to testSubQueryRemoveRuleForSubqueryWithInClause

Reply:
Changed as mentioned